controllers/api/userRoutes.js

```diff
@@ -6,27 +6,56 @@ const { Person } = require('../../models');
 // create a new user
 router.post('/', async (req, res) => {
     try {
-        
+        const newPerson = await Person.create({
+            username: req.body.username,
+            password: req.body.password,
+        });
+
+        req.session.save(() => {
+            req.session.loggedIn = true;
+            res.json(newPerson);
+        })
     } catch (error) {
-        
+        console.log(error);
+        res.status(500).json(error);
     }
 });
 
 // login
 router.post('/login', async (req, res) => {
     try {
-        
+        const personLoggingIn = await Person.findOne({
+            where: {
```

Comment:
I think we should include attribute selector to only select user id and username here. Because this will return hashed password, which I think we should avoid.